src/lib/pubkey/rsa/rsa.cpp

```diff
@@ -445,6 +445,20 @@ class RSA_Private_Operation {
             m_max_d1_bits(m_private->p_bits() + m_blinding_bits),
             m_max_d2_bits(m_private->q_bits() + m_blinding_bits) {}
 
+      void raw_op(std::span<const uint8_t> input, std::span<uint8_t> out) {
```

Comment:
Not sure this is a consistent convention, but typically, we put out-params first in the list.